app/controllers/admin/booths_controller.rb

```diff
@@ -0,0 +1,82 @@
+module Admin
+  class Admin::BoothsController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :booth, through: :conference
+
+
+    def index
+      @booths = @conference.booths
+    end
+
+    def show; end
+
+    def new
+      @booth = Booth.new(conference: @conference)
+    end
+
+    def create
+      @booth = @conference.booths.build(booth_params)
+
+      if @booth.save
+        redirect_to admin_conference_booths_path,
+                    notice: "Booth successfully created."
+      else
+        flash[:error] = "Creating booth failed. #{@booth.errors.full_messages.join('. ')}."
+        render :new
+      end
+    end
+
+    def edit; end
+
+    def update
+      @booth = @conference.booths
```

Comment:
This is very wrong. Also `@booth` is set by cancancan, you don't have to set it.